Dynatrace/dynatrace-operator
# Removes creation of StatsD (and consequently EEC) containers in ActiveGate pods

# Description
Removes the creation of StatsD and EEC pods and all related config options from the operator code and hence from E2E tests.

## How can this be tested?
Build and deploy the operator. Deploy a DynaKube with AG capability `statsd-ingest` activated should give you an error message.

## Checklist
